tools/scripts/generic_variables.mk

```diff
@@ -0,0 +1,24 @@
+# Create variables that will be used within src.mk
+
+# This variables will be set to their default value if not set by the user
+# It assumes that the makefile if invoked from No-OS/project/some_project
```

Comment:
nitpick: "makefile if->is invoked" I think you meant.

Reply:
done